aws-opensearchserverless-lifecyclepolicy/aws-opensearchserverless-lifecyclepolicy.json

```diff
@@ -0,0 +1,99 @@
+{
+  "typeName": "AWS::OpenSearchServerless::LifecyclePolicy",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-opensearchserverless",
+  "description": "Amazon OpenSearchServerless lifecycle policy resource",
+  "tagging": {
+    "taggable": false,
+    "tagOnCreate": false,
+    "tagUpdatable": false,
+    "cloudFormationSystemTags": false
+  },
+  "definitions": {
+    "LifecyclePolicyType": {
+      "type": "string",
+      "description": "The possible types for the lifecycle policy",
```

Comment:
Are these texts reviewed by documentation team?

Reply:
The texts are exactly same as other policies.